### PR TITLE
feature/banner and style

### PR DESCRIPTION
**Pull request by Kaival-Infynno**

### CODE REVIEW

File: src/Form/Button.jsx [hunk 1]
```diff
@@ -9,11 +9,11 @@ const Button = ({ variant, sm, lg, rounded, disabled, block }) => {
...
-          variant === "secondary" && "bg-gray-600",
-          variant === "dark" && "bg-black ",
-          variant === "success" && "bg-green-600",
-          variant === "alert" && "bg-red-600",
-          variant === "warning" && "bg-yellow-600",
+          variant === "secondary" && "bg-gray-600 text-white",
+          variant === "dark" && "bg-black text-white",
+          variant === "success" && "bg-green-600 text-white",
+          variant === "alert" && "bg-red-600 text-white",
+          variant === "warning" && "bg-yellow-600 text-white",
```
Comment (imkrunal) [inline src/Form/Button.jsx:16]: We should add some hover effects to buttons.

File: src/Home.jsx [hunk 2]
```diff
@@ -1,65 +1,131 @@
...
+            </span>
+          </div>
+        </div>
+        <div className="flex gap-8">
+          <FaRegFileCode className="h-16 w-16 text-blue-600" />
+          <div className="flex flex-col gap-2">
+            <span className="font-semibold"> React, Vue and HTML </span>
+            <span>
+              Beautifully designed, expertly crafted components that follow all
+              accessibility best practices and are easy to customize.
```
Comment (imkrunal) [inline src/Home.jsx:116]: Remove commented code.

File: src/Home/Introduction.jsx [hunk 3]
```diff
@@ -1,48 +1,58 @@
...
-        <div>
-          <div className="text-5xl font-bold">
-            Beautiful UI components, crafted with Tailwind CSS.
+    <div className="relative pt-[410px] h-0 bg-black">
+      <div className="absolute  top-0 left-0 right-0 bottom-0 opacity-40">
+        <img
+          src={banner}
+          alt="banner not found"
+          className="w-full object-cover h-[410px]"
+        />
```
Comment (imkrunal) [inline src/Home/Introduction.jsx:13]: Use next-image

**Review by imkrunal — CHANGES_REQUESTED** [review]

File: src/Form/Button.jsx (hunk 1 above)
Comment (Kaival-Infynno) [reply src/Form/Button.jsx:16]: Added ...

**Pull request merged**